client/src/catalog-element.html

```diff
@@ -722,23 +722,59 @@ <h1 hidden="[[subElement]]">README.md</h1>
           this._starRepo();
       },
 
+      _useAnalyzer: function(queryParams) {
+        return 'use_analyzer_data' in queryParams ? 'use_analyzer_data' : '';
+      },
+
       _shouldShowEditor: function(queryParams, prefix) {
         return 'editing' in queryParams || prefix == '/preview';
       },
 
       _elements: function() {
         if (!this.docs || !this.docs.content)
           return [];
-        return Object.keys(this.docs.content.elementsByTagName).map(function(elem) {
-          return this.docs.content.elementsByTagName[elem];
+
+        var content = this.docs.content.analyzerData || this.docs.content;
```

Comment:
So is it always going to be `docs.content.analyzerData` moving forward? Would it make more sense if this was `docs.docs` or `docs.analysis` in the API?